### ISSUE TITLE
Co-authors not recognised in Datatracker when an editor is appointed

### ISSUE THREAD

**Issue by JayDaley** [rfcseries-wg/new-topics#31]
## Problem Statement
1. There has to be a hard limit on the number of people who can participate as "authors" in the AUTH48 process, otherwise that process is unmanageable (because people disagree with each other).  This limit is generally considered to be five.
2.  When an editor is appointed in order to manage an excessive number of authors, the people who would otherwise be considered authors are no longer specified as such in the RFCXML and instead manually listed as "co-authors" in the Contributors section.  This means that they are not recognised as authors in Datatracker.
3.  An unlimited number of authors has been known to lead to people who are not actually authors, such as senior managers or supervisors, being added as authors despite not being contributors, and in some cases not even knowing they have been added.
## Options
1. Change the behaviour of xml2rfc so that people listed as Authors, when there is an Editor, are automatically listed in the Contributors section as "co-authors" rather than at the top of the document.  This would mean that they are recognised as authors in Datatracker.  This would also mean that they have no role in Auth48, that is solely for the Editor(s).   This enables strict enforcement of the maximum of five authors without any deleterious effects on those who authors who are not the editor.
2. Set a maximum number of people who can answer for a document in AUTH48 and if the number of authors exceeds that then the authors (or WG) need to decide who is on the list and who isn’t before AUTH48 can start.  This enables removing the maximum of five authors.
3. No longer list any person at the top of the document and instead automatically list all the authors (and editor) in the Contributors section. This enables removing the maximum of five authors.

**Comment by mnot** [comment]
> This means that they are not recognised as authors in Datatracker.

This seems to motivate the issue, but the impact isn't spelled out. Why is it an issue if they aren't recognised as authors in Datatracker?

**Comment by JayDaley** [comment]
People expect to see a record of their authorship in Datatracker and are unhappy if they "lose" authorship recognition when an Editor is appointed, having had that recognition for the I-Ds that preceded.  This loss of recognition is a driver for people wanting more than five authors on a document, which causes ongoing issues.

**Comment by ekr** [comment]
This issue seems to make some assumptions that are not strictly true.

1. There is not in a formal hard limit on the number of authors. This is a stream decision, with five as just the "general" number, as clearly stated in RFC 7322, S. 4.1.1. 
2. While it is sometimes the case that an editor is appointed and is the only person whose name is listed on the top of the document, in other cases an editor is named but other people still appear on the top of the document. See, for instance RFC 6940. In my experience, the role of "Editor" is a somewhat flexible one.

In terms of substance, I do agree that it's not really that helpful to have a large number of people have to sign off on AUTH48, though I don't think there has to be a hard limit. I do think it would be potentially helpful to have a notion of "authors to whom correspondence should be addressed" and who are responsible for the final approval, but I don't see any reason why this has to be reflected on the front page of the document, as it's a procedural question. Similarly, the question of what's in the datatracker seems like the tail wagging the dog. If the datatracker doesn't currently accommodate the concept of who is actually responsible for AUTH48, it can be changed to do so. What matters is who appears in the author list in the document proper.

It seems like decoupling the role of Author from AUTH48 approver would require a change to RFC 7322. As noted above, I'm not opposed to that, but it's also not clear to me that this occurs often enough to make it a high priority. Do you have some data?

**Comment by JayDaley** [comment]
The 'hard limit" in the problem statement relates to ATUH48 not the rules about documents.  I think that is clear in the text.

Yes there can be multiple authors with one or more labelled as editors and they all appear at the top, but this is about the specific case where an editor is appointed and those who were previously authors are no longer listed as such.  That does need to be clearer in the text.

Option 2 addresses your preferred approach - all it does is decouple the AUTH48 participants from the list of authors, it says nothing about where they appear.

I agree we do not have enough data to establish a priority here.

**Comment by ekr** [comment]
> The 'hard limit" in the problem statement relates to ATUH48 not the rules about documents. I think that is clear in the text.

No, this isn't particularly clear, especially as they are coupled in 7322. Regardless, I don't agree with your assertion that there has to be a hard limit, as opposed to IESG discretion. So, no, what I propose isn't what you propose in option (2), because I don't think we need a maximum limit, but rather the ability to decouple these if/when the IESG deems it necessary.
